psng/python/tool_measurement.py

```diff
@@ -83,8 +83,7 @@ def _init_tool_sensor_data(self):
             or tsdiam is None
         ):
             self.chk_use_tool_measurement.set_active(False)
-            self.btn_tool_dia.set_sensitive(False)
-            self.btn_probe_tool_setter.set_sensitive(False)
+            self.frm_probe_pos.set_sensitive(False)
```

Comment:
So, these can actually be removed now - since the `error_dialog()` method was added, if the INI is invalid, PSNG will show the message and then quit - so there's no need to disable the form.

I'll submit a PR that cleans the whole active/sensitive stuff in here up.